src/utils/openInBrowser.ts

```diff
@@ -0,0 +1,10 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+
+export async function openInBrowser(url: string): Promise<void> {
```

Comment:
Consider adding an assert that the URL starts with https

Reply:
Are you worried about http vs https? If so, we already have a tslint rule "no-http-string"

Or are you worried about non-http/https? If so, I'd prefer not to restrict this. I figure VS Code already has validation under the covers and people might use this for other urls in the future. I've renamed from `openInBrowser` to `openUrl` to reflect that